### PR TITLE
refactor: disable selecting non-yaml files

### PR DESCRIPTION
**Pull request by catalinhoha**
This PR...

## Changes

- disables non-yaml file selections

## Fixes

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/components/organisms/FileTreePane/FileTreePane.tsx [hunk 1]
```diff
@@ -309,6 +309,9 @@ const FileTreePane = () => {
   }, [selectedPath]);
 
   const onSelect = (selectedKeysValue: React.Key[], info: any) => {
+    if (!['.yaml', '.yml'].includes(path.extname(info.node.key))) {
```
Comment (olensmar) [inline src/components/organisms/FileTreePane/FileTreePane.tsx:312]: you could check against the appConfig.fileIncludes array?

Comment (catalinhoha) [reply src/components/organisms/FileTreePane/FileTreePane.tsx:312]: but then the user could include other file types that can be selected and shown in the editor, do we want that for now?

Comment (olensmar) [reply src/components/organisms/FileTreePane/FileTreePane.tsx:312]: those files would be parsed for manifests - so I think it's ok to make them editable also

Comment (catalinhoha) [reply src/components/organisms/FileTreePane/FileTreePane.tsx:312]: I see, okay, will modify it

**Review by olensmar — APPROVED** [review]

**Pull request merged**